src/qfusion.cpp

```diff
@@ -0,0 +1,541 @@
+//////////////////////////////////////////////////////////////////////////////////////
+//
+// (C) Daniel Strano and the Qrack contributors 2017, 2018. All rights reserved.
+//
+// QUnitMulti is a multiprocessor variant of QUnit.
```

Comment:
Probably need to update this comment too.

Reply:
Done.